.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+
+## Description
+
+<!--
+Please describe your change and provide as much context as needed including a link to the reference issue / ticket / Trello card. 
+-->
+
+* How to test the change: <description or reference to the Acceptance Criteria>
+* Related Issue / Ticket / Trello card: <link reference>
+
+## Review
+
+Please check out our [review guidelines](https://github.com/SUSE/scc-docs/blob/master/team/workflow/code_review.md) 
```

Comment:
Again, this is not available for external contributions.

Reply:
I know, but 99% of pull requests are internal, and it was requested in the dev sync to have the reference to our review guidelines in all our projects.